defaults/main.yml

```diff
@@ -330,11 +333,18 @@ nginx_ocsp_resolvers: []
 
 # HTTP Strict-Transport-Security
 # https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security
+# Maximum age in seconds for which clients should remember to only make secure
+# connections.
+# Defaults to six earth month.
```

Comment:
No, it's fine, although I would write it as "Defaults to six Earth months." I guess it's an issue with source material.